launcher_scripts/conf/training/gpt3/126m.yaml

```diff
@@ -109,14 +109,16 @@ model:
   grad_allreduce_chunk_size_mb: 125
 
   ## Transformer Engine
-  transformer_engine: False
+  # To use fp8, change only `fp8` to `True`.
+  # The rest of fp8 knobs are set for the fp8 training mode, which are ignored in non-fp8 training
+  transformer_engine: True
```

Comment:
Can we revert this to `False` as `True` is currently breaking CI until we fix TE + PEFT

Reply:
Done

Reply:
It's still True :D

Reply:
N/m we will update the CI